ProcessLib/ComponentTransport/ComponentTransportFEM.h

```diff
@@ -275,9 +287,9 @@ class LocalAssemblerData : public ComponentTransportLocalAssemblerInterface
         pos.setElementID(_element.getID());
 
         MaterialLib::Fluid::FluidProperty::ArrayType vars;
-
-        auto const p_nodal_values = Eigen::Map<const NodalVectorType>(
-            &local_x[ShapeFunction::NPOINTS], ShapeFunction::NPOINTS);
+        auto const num_nodes = ShapeFunction::NPOINTS;
```

Comment:
@TomFischer Seems there is no difference (in the optimized assembly) whether to use NPOINTS or num_nodes for the Eigen::Map creation.